test/powershell/Modules/Microsoft.PowerShell.Management/Restart-Computer.Tests.ps1

```diff
@@ -124,6 +124,6 @@ Describe 'Non-admin on Unix' {
 
     It 'Reports error if not run under sudo' -Skip:($IsWindowsOrSudo) {
         $error.Clear()
-        { Restart-Computer -ErrorAction Stop } | Should -Throw -ErrorId "CommandFailed,Microsoft.PowerShell.Commands.RestartComputerCommand" -Because (Get-Error)
+        { Restart-Computer -ErrorAction Stop } | Should -Throw -ErrorId "CommandFailed,Microsoft.PowerShell.Commands.RestartComputerCommand" -Because (Get-Error | Out-String)
```

Comment:
It seems we should use try-catch if we want to catch the exception for debug.

Reply:
I may have to try that as the current mechanism with `Should` doesn't show it for some reason

Reply:
I think I figured out the problem, the cmdlet is hardcoded to use `/sbin/shutdown`, but the container image probably has a different path.  My WSL instance of Ubuntu uses `/usr/sbin/shutdown`.  So I'll update the cmdlet to check both locations for the command.  The `win32exception` is probably coming from starting the process.  Still not sure why I can't capture the error and stacktrace in the pester test which would make this easier to debug.

Reply:
> Still not sure why I can't capture the error and stacktrace in the pester test which would make this easier to debug.

Perhaps it is Pester limitation for the exception kind.
/cc @nohwnd